view/templates/admin/blocklist/server.tpl

```diff
@@ -4,20 +4,23 @@
 	}
 </script>
 <div id="adminpage">
-	<h1>{{$title}} - {{$page}}</h1>
+	<h2>{{$title}} - {{$page}}</h2>
```

Comment:
Why have you lowered the header level? It is the first header on the page, so from the semantic it also should be a h1 level.

Reply:
Because this isn't the first header on the page, or at least I don't think it should be. The first header on the page should be something like `<h1>Friendica</h1>` at the start of the page. It doesn't currently exist in the frio theme, but I think it should, even if it is hidden with CSS.

After a cursory search it doesn't even seem to be currently consistent across all Friendica's  templates.

Reply:
The "Friendica" is the title of the page (or header zero), and the 1st header on the page is "Administration Server Blocklist"

Reply:
But we do show the Friendica logo in the body (not only in the title) and I think it should be the first header.

Reply:
The logo is part of the menu and it makes no sense to mark the menu as header. If there was no menu at the top of the page then your argument might make sense depending on the layout of the page, but not with the menu at top.

Reply:
I reverted this change until we agree on a definitive HTML header tags policy for Friendica.

Reply:
FYI, this is the look of the Friendica Admin Server Block page without CSS:
![2019-07-26 22 11 09 dev-friendica mrpetovan com 50a466f2b661](https://user-images.githubusercontent.com/925415/61988671-6d2b2000-aff2-11e9-895e-f3415176131a.png)
This isn't personally your fault, of course, but I think a page should be legible and usable even without CSS.

Reply:
Just to explain why it is so important to number the headers correctly: Some time ago I spoke with blind people who are using screenreaders. They are easily confused when they read a page and there are gaps in the header numbering. We do have some flaws in our design that currently are making Friendica not that accessible as I would like to see it.

I hope that some day we will find some developer who is passionate enough to spend some time for this.

Reply:
@MrPetovan off topic: does your domain block in your screenshot work at all? I always thought one must put a domain into the field, not a URL.

Reply:
No, but it's on my dev node, so I haven't actually checked if the block was working because I don't care. You're absolutely right, it should be only domains.